### PR TITLE
Release 4.2.0

### PR DESCRIPTION
**Pull request by ashie**

### CODE REVIEW

**Comment by kenhys** [comment]
TODO: ensure about https://github.com/fluent-plugins-nursery/td-agent-builder/issues/305#issuecomment-882402626

**Comment by kenhys** [comment]
TODO: check CentOS 7 aarch64 again.

**Comment by kenhys** [comment]
Fixed fluentd version.

**Comment by kenhys** [comment]
Fix missing updated gems (rdkafka, fluent-plugin-prometheus_pushgateway) wrong version (fluent-plugin-utmpx)

**Comment by kenhys** [comment]
> TODO: check CentOS 7 aarch64 again.

local build succeeds.

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,20 +38,21 @@
...
+* fluent-plugin-prometheus_pushgateway v0.1.0 (update)
 * systemd-journal v1.4.2 (update)
 * fluent-plugin-systemd v1.0.5 (update)
 * fluent-plugin-record-modifier v2.1.0
 * fluent-plugin-rewrite-tag-filter v2.4.0
 * fluent-plugin-sd-dns 0.1.0
 * fluent-plugin-flowcounter-simple 0.1.0
 * fluent-diagtool v1.0.1
-* fluent-plugin-utmpx v0.3.0
+* fluent-plugin-utmpx v0.5.0
```
Comment (ashie) [inline CHANGELOG.md:55]: > (Maybe confused with linux-utmpx?)

Yes, that's right. Thanks following up!

**Comment by kenhys** [comment]
Surely dependency to perl was removed.
```
% rpm -qp --requires ./td-agent/yum/repositories/amazon/2/x86_64/Packages/td-agent-4.2.0-1.amzn2.x86_64.rpm |\grep perl
```

**Comment by kenhys** [comment]
Added more explanation about jeprof

**Comment by kenhys** [comment]
list of contained files (compared to 4.1.1) seems ok. (removed bundler/gems/fluentd-xxx, removed jeprof)
[debdiff.txt](https://github.com/fluent-plugins-nursery/td-agent-builder/files/6890666/debdiff.txt)
[rpmdiff.txt](https://github.com/fluent-plugins-nursery/td-agent-builder/files/6890667/rpmdiff.txt)

**Comment by ashie** [comment]
We are going to upload built packages.
It may take several days to be accepted.

**Pull request merged**